src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -130,7 +129,7 @@ public boolean isRegistered() {
     }
 
     public String getRegistrationUrl() {
-        return new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+        return new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
```

Comment:
For the very same reason, take a look at the changes I made in `PageData.java`: I believe that they're unnecessary, but I want to keep the diff minimum for now (otherwise I'm changing a few other HTML files; or should I go down this path?).